Userland/Services/WebServer/Client.cpp

```diff
@@ -226,9 +226,9 @@ static String folder_image_data()
 {
     static String cache;
     if (cache.is_empty()) {
-        auto file_or_error = Core::MappedFile::map("/res/icons/16x16/filetype-folder.png");
-        VERIFY(!file_or_error.is_error());
-        cache = encode_base64(file_or_error.value()->bytes());
+        // FIXME: Propagate errors
+        auto file = MUST(Core::MappedFile::map("/res/icons/16x16/filetype-folder.png"));
```

Comment:
errors have a `release_value_but_fixme_should_propagate_errors()` (or something like that) function for this use case

Reply:
Fixed